policyengine_us/parameters/gov/states/ma/tax/income/credits/dependent/amount.yaml

```
@@ -1,12 +1,19 @@
 description: MA dependent credit amount.
 values:
   2021-01-01: 180
+  2023-01-01: 310
```

This isn't in the bill you cited, so please cite this one instead which Governor Healey signed: https://malegislature.gov/Bills/193/H4104

Note the 2024 change occurs in Section 22, and Section 45 indicates that it takes effect in 2024.
```suggestion
  2023-01-01: 310
  2024-01-01: 440
```